apps/services/user-notification/src/app/modules/notifications/me-notifications.controller.ts

```diff
@@ -100,6 +100,17 @@ export class MeNotificationsController {
     await this.notificationService.markAllAsSeen(user)
   }
 
+  @Patch('/mark-all-as-read')
```

Comment:
We have a described RPC like endpoints in our API Design Guide and the should be using `POST`, `mark-all-as-seen` above as well.